code/drasil-code/lib/Language/Drasil/CodeSpec.hs

```diff
@@ -70,7 +67,9 @@ data CodeSpec where
   -- automatically define.
   mods :: [Mod],  -- medium hack
   -- | The database of all chunks used in the problem.
-  sysinfodb :: ChunkDB
+  sysinfodb :: ChunkDB,
```

Comment:
I thought the aim was to rename this `CodeSpec` to `OldCodeSpec` and then make `CodeSpec` has just 2 fields?